Resources/config/csrf.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <service id="sensio_framework_extra.security.listener" class="Sensio\Bundle\FrameworkExtraBundle\EventListener\CsrfListener">
+            <argument type="service" id="security.csrf.token_manager" on-invalid="null" />
```

Comment:
Why not simply removing this definition if the `security.csrf.token_manager` service is not available?